Drivers/seds/AfecHwas/example/work/manager/implem/default/C/src/manager.c

```diff
@@ -0,0 +1,36 @@
+/* Body file for function Manager
+ * Generated by TASTE on 2022-04-06 11:12:12
+ * You can edit this file, it will not be overwritten
+ * Provided interfaces : GetTemperature
+ * Required interfaces : AfecHwasInterface_GetValue_Pi, AfecHwasInterface_InitInstance_Pi
+ * User-defined properties for this function:
+ * Timers              : 
+ */
+
+#include "manager.h"
+//#include <stdio.h>
```

Comment:
Remove

Reply:
Ok, I think this is automatically added by space-creator. (this is a part of the template)

Reply:
done